File: src/main/java/com/meti/Return.java
package com.meti;

record Return(Token value) implements Token {
	@Override
	public Attribute apply(Attribute.Name name) throws AttributeException {
		return switch (name) {
			case Group -> Group.Return;
			case Value -> new TokenAttribute(value);
			default -> throw new AttributeException();
		};
	}
}

File: src/main/java/com/meti/Return.scala
package com.meti

import com.meti.Attribute.Name
import com.meti.Attribute.Name._

final class Return(val value: Token) extends Token {
  @throws[AttributeException]
  override def apply(name: Name): Attribute = name match {
    case Group => Token.Group.Returns
    case Value => TokenAttribute(value)
    case _ => throw AttributeException()
  }
}